addons/binding/org.openhab.binding.gpstracker/src/main/java/org/openhab/binding/gpstracker/internal/handler/TrackerHandler.java

```diff
@@ -186,7 +217,9 @@ private void mapDistanceChannels() {
 
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
-        //no command handling
+        if (command instanceof RefreshType && lastMessage != null) {
+            updateLocation(lastMessage);
```

Comment:
The not so nice part about this is that you will receive a REFRESH command PER CHANNEL. So you are actually now updating all channels multiple times 🤔.

Reply:
Hm... It's not that simple... In this case I'll need some refactoring for this...

Reply:
Yeah, sorry that I haven't noticed that earlier...

Reply:
Should be OK now.